src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -0,0 +1,167 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React, {useEffect, useMemo, useState} from 'react';
+import {FlatList, View} from 'react-native';
+import * as Illustrations from '@components/Icon/Illustrations';
+import MenuItem from '@components/MenuItem';
+import Section from '@components/Section';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useNetwork from '@hooks/useNetwork';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import * as OptionsListUtils from '@libs/OptionsListUtils';
+import * as ReportUtils from '@libs/ReportUtils';
+import type {CentralPaneNavigatorParamList} from '@navigation/types';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import WorkspacePageWithSections from '@pages/workspace/WorkspacePageWithSections';
+import * as Policy from '@userActions/Policy';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+import ToggleSettingOptionRow from './ToggleSettingsOptionRow';
+import type {ToggleSettingOptionRowProps} from './ToggleSettingsOptionRow';
+
+type WorkspaceWorkflowsPageProps = WithPolicyProps & StackScreenProps<CentralPaneNavigatorParamList, typeof SCREENS.WORKSPACE.WORKFLOWS>;
+
+function WorkspaceWorkflowsPage({policy, route}: WorkspaceWorkflowsPageProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const StyleUtils = useStyleUtils();
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const {isOffline} = useNetwork();
+
+    const [policyOwnerDisplayName, setPolicyOwnerDisplayName] = useState('');
```

Comment:
I'm not sure this state is necessary. We're only updating it in the useEffect below with `policy` in the dependency array, but when policy changes the whole component already re-renders, so it seems redundant to have it in state.

Reply:
We need it to calculate the display name of the policy owner, and as such updating state is necessary when we finally retrieve it.

Otherwise we only have the policy's owner email or owner account ID available in Onyx, which is not what we want?

We need to calculate it, if you can propose another solution feel free to tell me 😄

Reply:
I think that's what I'm saying. We can calculate the value without storing it in state since the component would re-render when the policy object updates, we can calculate the value then.